config-htsget/config.json

```
@@ -0,0 +1,37 @@
+{
+  "htsgetConfig": {
+    "props": {
+      "port": "3000",
+      "host": "http://localhost/",
+      "logFile": "/logs/htsget.log"
+    },
+    "reads": {
+      "enabled": true,
+      "dataSourceRegistry": {
+        "sources": [
+          {
+            "pattern": "^local/(?P<accession>.*)$",
+            "path": "/usr/src/app/data/gcp/gatk-test-data/{accession}.bam"
+          },
```

```suggestion
          {
            "pattern": "^local/(?P<accession>.*)$",
            "path": "/usr/src/app/data/gcp/gatk-test-data/wgs_bam/{accession}.bam"
          },
```

Those files should probably be removed from the htsget repo